### PR TITLE
[Feature] : Navbar mobile site changed

### LINKED ISSUE #258: [Feature] : Navbar mobile site
Hello @Tanay-ErrorCode 
-  In this project `Nav bar` mobile site it's like so simple.
- In this `Navbar` it so small .

### Actual work
 
Screenshot from 2024-06-23 08-27-48

### My work

[Screencast from 2024-06-23 08-25-53.webm](https://github.com/Tanay-ErrorCode/lupo-skill/assets/121082414/a5241254-6f7b-4fea-b22d-77cfff51b597)

### PR DESCRIPTION
**Pull request by sivaprasath2004**
Hello @Tanay-ErrorCode ,
**What kind of change does this PR introduce?**

[Feature] : Navbar mobile site  

**Issue Number:**

Fixes #258 
**Snapshots/Videos:**

### Before

 
Screenshot from 2024-06-23 08-27-48

### After

[Screencast from 2024-06-24 09-31-35.webm](https://github.com/Tanay-ErrorCode/lupo-skill/assets/121082414/300e2877-14f7-4542-9334-8aa5b01c99cc)

### CODE REVIEW

File: src/firebaseConf.ts [hunk 1]
```diff
@@ -1,60 +0,0 @@
-import {
```
Comment (Doraemon012) [inline src/firebaseConf.ts:1]: Why did you delete this file ?

File: src/components/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -628,10 +605,21 @@ const NavBar = () => {
...
-                  {mobileMenuItems}
-                </Box>
+                <Drawer
+                  anchor="top"
+                  open={true}
+                  onClose={() => setExpanded(false)}
+                >
+                  <Box
+                    sx={{
+                      backgroundColor: "#0F0F0F",
```
Comment (Doraemon012) [inline src/components/NavBar/NavBar.tsx:615]: Use the colors from `App.css` or `Theme.tsx`. Do not hard code the values.

File: src/components/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -628,10 +605,21 @@ const NavBar = () => {
                   },
                 }}
               >
-                {" "}
-                <Box sx={{ backgroundColor: "#0F0F0F", width: "100%" }}>
-                  {mobileMenuItems}
-                </Box>
+                <Drawer
+                  anchor="top"
```
Comment (Doraemon012) [inline src/components/NavBar/NavBar.tsx:609]: > anchor="top"
It should be right.

File: src/components/NavBar/NavBar.tsx [hunk 4]
```diff
@@ -277,7 +254,7 @@ const NavBar = () => {
                     : theme.fontSize.subheading,
                 }}
               >
-                Lupo Skill
+                Drawer Lupo Skill
```
Comment (Doraemon012) [inline src/components/NavBar/NavBar.tsx:257]: Why did you add this ?

**Comment by Doraemon012** [comment]
@sivaprasath2004 
Do not add icons.

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Make the requested changes.

**Comment by sivaprasath2004** [comment]
@Doraemon012 please check

**Comment by Tanay-ErrorCode** [comment]
@sivaprasath2004 Any updates?

**Comment by sivaprasath2004** [comment]
[Screencast from 2024-06-27 09-44-30.webm](https://github.com/Tanay-ErrorCode/lupo-skill/assets/121082414/dfff5bf9-5174-4bad-97ec-5fe1ec4fed90)

Hello @Tanay-ErrorCode , i will changed it.

**Review by Doraemon012 — APPROVED** [review]

**Comment by Tanay-ErrorCode** [comment]
@sivaprasath2004 Good work.

**Pull request merged**